### PR TITLE
feat: adds the v1 API surface

### PR DESCRIPTION
**Pull request by bcoe**
runs the generator with the v1 API surface.

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -11,7 +11,7 @@ jobs:
       matrix:
         node: [8, 10, 12, 13]
     steps:
-      - uses: actions/checkout@v1
+      - uses: actions/checkout@v2
```
Comment (JustinBeckwith) [inline .github/workflows/ci.yaml]: Could I trouble you for a hammer PR on this one?  We shouldn't be rename/merging this 80 times :)

File: src/index.ts [hunk 2]
```diff
@@ -17,10 +17,11 @@
...
 
 import * as v1beta1 from './v1beta1';
+import * as v1 from './v1';
 const WebRiskServiceV1Beta1Client = v1beta1.WebRiskServiceV1Beta1Client;
-export {v1beta1, WebRiskServiceV1Beta1Client};
+export {v1, v1beta1, WebRiskServiceV1Beta1Client};
 // For compatibility with JavaScript libraries we need to provide this default export:
 // tslint:disable-next-line no-default-export
-export default {v1beta1, WebRiskServiceV1Beta1Client};
+export default {v1, v1beta1, WebRiskServiceV1Beta1Client};
```
Comment (JustinBeckwith) [inline src/index.ts]: I don't understand what this line of code does.  Do you have a good grip on why/if this is needed?

File: .github/workflows/ci.yaml (hunk 1 above)
Comment (bcoe) [reply .github/workflows/ci.yaml]: this PR has already gone out to all repositories, and is the one Alex is working on landing as we speak.

File: src/index.ts (hunk 2 above)
Comment (bcoe) [reply src/index.ts]: This approach is necessary to ensure that the module has the same export signature as the JavaScript library did prior to the TypeScript conversion:

```bash
{
  TextToSpeechClient: [Function: TextToSpeechClient],
  v1beta1: { TextToSpeechClient: [Function: TextToSpeechClient] },
  v1: [Circular],
  default: {
    TextToSpeechClient: [Function: TextToSpeechClient],
    v1beta1: { TextToSpeechClient: [Function: TextToSpeechClient] },
    v1: [Circular]
  }
}
```

**Comment by bcoe** [comment]
the v1 and v1beta surfaces are identical, I learned talking with @bsurmanski. So I've updated samples accordingly.

**Pull request merged**